etip/trackers/tests.py

```diff
@@ -321,3 +324,155 @@ def test_with_trackers(self):
             ]
         }
         self.assertJSONEqual(f.getvalue(), expected_json)
+
+
+EXODUS_API_BASE_URL = 'https://reports.exodus-privacy.eu.org'
+EXODUS_API_BASE_PATH = '/api/trackers'
+
+
+class CompareTrackersWithExodusCommandTest(TestCase):
+
+    def test_api_gets_called_correctly(self):
+        with patch('requests.get') as mocked_get:
+            mocked_get.return_value.status_code = 200
+            out = StringIO()
+            call_command('compare_with_exodus', stdout=out)
+        mocked_get.assert_called_with(
+            EXODUS_API_BASE_URL + EXODUS_API_BASE_PATH
+        )
+
+    def test_raise_error_when_api_not_200(self):
+        with patch('requests.get') as mocked_get:
+            mocked_get.return_value.status_code = 404
+            with self.assertRaises(CommandError) as e:
+                out = StringIO()
+                call_command('compare_with_exodus', stdout=out)
+            error_msg = str(e.exception)
+            self.assertEqual(error_msg, 'Unexpected status from API: 404')
+
+    def test_raise_error_when_empty_response(self):
+        with patch('requests.get') as mocked_get:
+            mocked_get.return_value.status_code = 200
+            mocked_get.return_value.json.return_value = {}
+            with self.assertRaises(CommandError) as e:
+                out = StringIO()
+                call_command('compare_with_exodus', stdout=out)
+            error_msg = str(e.exception)
+            self.assertEqual(error_msg, 'Empty response')
+
+    def test_select_only_in_exodus_trackers(self):
+        with patch('requests.get') as mocked_get:
+            mocked_get.return_value.status_code = 200
+            Tracker(name="In Exodus", is_in_exodus=True).save()
```

Comment:
I would move the Trackers creation:
- outside of the mock 
- in the beginning of the test function

Reply:
Done! Refactored & added missing tests